data/sql/updates/pending_db_world/rev_1723083294200919589.sql

```diff
@@ -0,0 +1,9 @@
+-- Bug Report #19575
+-- Update required levels for:
+-- 3638 (Goblin Engineering - Pledge of Secrecy)
+UPDATE `quest_template_addon` SET `MinLevel`=20 WHERE `ID`=3638;
+-- 3640 (Gnome Engineering - Pledge of Secrecy, Alliance)
+UPDATE `quest_template_addon` SET `MinLevel`=20 WHERE `ID`=3640;
+-- 3642 (Gnome Engineering - Pledge of Secrecy, Horde)
+UPDATE `quest_template_addon` SET `MinLevel`=20 WHERE `ID`=3642;
```

Comment:
None of these quests exist in a 51831 brute.

Reply:
Aye, my friend and I (who discovered the issue) were debating removing the quest or just adjusting the level requirements, as both are arguable from the information we uncovered.

Reply:
Updated to remove Pledge of Secrecy from the chain instead